task1/task1_2.js

```diff
@@ -0,0 +1,29 @@
+import { createReadStream, createWriteStream } from 'fs';
+import { Transform } from 'stream';
+import { createInterface } from 'readline';
+import csvtojson from 'csvtojson';
+
+const csvFolderPath = './task1/csv';
+const csvFileName = 'books.csv';
+const csvFilePath = `${csvFolderPath}/${csvFileName}`;
+const nowDate = new Date();
+const dateStr = nowDate.toISOString().replace(/:/g, '.');
```

Comment:
For what you make this?

Reply:
![image](https://user-images.githubusercontent.com/24764105/111347714-86358880-8690-11eb-8938-45e1aad86466.png)
One of the requirements is to write in new txt file. I understood it in a way that we need to create a new txt file every time we start the program

Reply:
I think what was meant is that you do not need to overwrite the csv file, but write to new as another file
Anyway I believe that [uuid](https://www.npmjs.com/package/uuid) will be better for generate unic value

Reply:
In my opinion, date string is better because it achieves two goals: uniquelity and readability :)